usecases/traverser/explorer_validate_scroll.go

```diff
@@ -0,0 +1,34 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package traverser
+
+import (
+	"fmt"
+
+	"github.com/weaviate/weaviate/entities/dto"
+	"github.com/weaviate/weaviate/entities/filters"
+	"github.com/weaviate/weaviate/entities/schema"
+)
+
+func (e *Explorer) validateScroll(params dto.GetParams) error {
+	if params.Scroll != nil {
+		if params.Group != nil || params.HybridSearch != nil || params.KeywordRanking != nil ||
+			params.NearObject != nil || params.NearVector != nil || len(params.ModuleParams) > 0 {
```

Comment:
shouldn't there be `nearText` somewhere?

Reply:
len of `len(params.ModuleParams)` will be `>0` if `nearText` is set, as `nearText` is only present on a setup with a `t2v` module.

Reply:
correct, `len(params.ModuleParams) > 0` covers this